### PR TITLE
Adjust sampleapp systemd file to new installation path

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: sample_app/pnet-sampleapp.service [hunk 1]
```diff
@@ -3,15 +3,18 @@
...
 [Service]
 Type=simple
+ExecStartPre=/sbin/ifconfig eth0 up
 ExecStartPre=/bin/sh -c '/bin/echo "49152 60999" > /proc/sys/net/ipv4/ip_local_port_range'
 ExecStartPre=/usr/bin/gpio export 22 in
 ExecStartPre=/usr/bin/gpio export 27 in
-WorkingDirectory=/home/pi/profinet/p-net/build/
-ExecStart=/home/pi/profinet/p-net/build/pn_dev -v -b /sys/class/gpio/gpio22/value -d /sys/class/gpio/gpio27/value
+WorkingDirectory=/home/pi/profinet/build/
+ExecStart=/home/pi/profinet/build/pn_dev -v -b /sys/class/gpio/gpio22/value -d /sys/class/gpio/gpio27/value
```
Comment (olbjo) [inline sample_app/pnet-sampleapp.service:15]: Maybe default should be to run without the real gpios?
Only with -v option. gpio version can be kept but commented out but available for advanced users.
This version works for my build/setup.

**Review by olbjo — APPROVED** [review]
Great!

File: sample_app/pnet-sampleapp.service (hunk 1 above)
Comment (pyhys) [reply sample_app/pnet-sampleapp.service:15]: I do not think using inputs is affecting anything negatively?

Comment (olbjo) [reply sample_app/pnet-sampleapp.service:15]: 👍

**Pull request merged**